src/features/import/utils/problems/predictProblems.ts

```diff
@@ -20,6 +20,7 @@ const VALIDATORS: Record<CUSTOM_FIELD_TYPE, (value: string) => boolean> = {
       return false;
     }
   },
+  enum_text: () => true,
```

Comment:
Is there anything actually testing this? I can't see that there's a test in `predictProblems.spec.ts` that tests this.

Reply:
Added a small test for this